internal/storage/serde_events_test.go

```diff
@@ -343,3 +343,56 @@ func TestDeltalogSerializeWriter(t *testing.T) {
 		}
 	})
 }
+
+func TestDeltalogV2(t *testing.T) {
```

Comment:
We shall add some benchmarks on new format's CPU and IO cost, so to better demonstrate the advantages of the new format.